src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements Calculate, Draw {
```

Comment:
Don't use abstract classes to set behavior for classes. An abstract class is generally used when there is a clear is-a relationship and some common behavior or state. However, for defining common behavior without state, interfaces are preferred. Consider using interfaces for `Calculate` and `Draw` if they don't share common state.